### ISSUE TITLE
Porting entire book

### ISSUE THREAD

**Issue by VRamazing** [shiffman/Learning-p5.js#3]
Hello I love your work.Can I contribute to this repo. I am following your book learning processing in java and it has been a bit difficult for me to code from java to js.Possibly many more are facing the same problem.Although it is great book still it would be better if we had a complete port in p5 js like nature of code.

**Comment by shiffman** [comment]
Yes, I would very much like to do this and chapter 8 is here as an experiment. However, I do not own the book in the same way as Nature of Code since I worked with a publisher. I am doing some research and thinking re: next steps and will remember to follow up here.

**Comment by VRamazing** [comment]
@shiffman  We can at least port the code if not the entire book to p5.js . It would be really helpful

**Comment by shiffman** [comment]
Here it is! https://github.com/shiffman/LearningProcessing-p5.js

But actually a lot of the examples are broken on learningprocessing.com (running with p5.js in the background). I could use help with this!

https://github.com/shiffman/learningprocessing.com/issues/39

**Comment by VRamazing** [comment]
Awesome I am on it.
